pkg/table/tables/tables.go

```diff
@@ -1368,7 +1368,7 @@ func (t *TableCommon) RemoveRecord(ctx table.MutateContext, h kv.Handle, r []typ
 	memBuffer.Release(sh)
 
 	if shouldWriteBinlog(ctx.GetSessionVars(), t.meta) {
-		cols := t.Cols()
+		cols := t.DeletableCols()
```

Comment:
Consistent with https://github.com/pingcap/tidb/blob/afd23434dc558f00514dd7cc0295807002b69e16/pkg/planner/core/logical_plan_builder.go#L4893-L4895

Reply:
when `add column` this list will contain the new column that `binlog` does not know now, can binlog handle it?

Reply:
Drainer can handle it correctly by only decoding necessary(public) columns:
https://github.com/pingcap/tidb-binlog/blob/6fba4f883a0c6c7e0d68b3b8aef1a53d9454b273/drainer/translator/mysql.go#L88-L90